### PR TITLE
N°6152 - Search: Criteria & object list loaded twice

### PR DESCRIPTION
**Pull request by accognet**
put 'submit_on_load' to false in each case

### CODE REVIEW

**Review by Molkobain — APPROVED** [review]

**Comment by Molkobain** [comment]
Accepted technical review

**Comment by piRGoif** [comment]
@accognet, you did push a commit since review, so we need to review it again ?

**Comment by Molkobain** [comment]
Accepted during functional review for 3.1 branch.
Last review to be done with Guillaume then merge.

File: js/links/links_widget.js [hunk 1]
```diff
@@ -153,16 +153,15 @@ function LinksWidget(id, sClass, sAttCode, iInputId, sSuffix, bDuplicates, oWizH
 				"dataType": "html"
 			})
 			.done(function (data) {
+				/*N°6152 - hide during data loading and before open*/
```
Comment (Molkobain) [inline js/links/links_widget.js]:
```suggestion
				/* N°6152 - Hide during data loading and before open */
```

**Review by Molkobain — APPROVED** [review]

**Pull request merged**